File: elixir/03-operators/arithmetic.exs
a = 17
b = 5

IO.puts("a + b = #{a + b}")
IO.puts("a - b = #{a - b}")
IO.puts("a * b = #{a * b}")
IO.puts("a / b = #{a / b}")
IO.puts("div(a, b) = #{div(a, b)}")
IO.puts("rem(a, b) = #{rem(a, b)}")

# Unary minus
IO.puts("-a = #{-a}")

# Exponentiation via Erlang's :math.pow (always returns a float)
IO.puts("2 ** 10 = #{:math.pow(2, 10)}")

File: julia/03-operators/arithmetic.jl
a = 17
b = 5

println("a + b = ", a + b)
println("a - b = ", a - b)
println("a * b = ", a * b)
println("a / b = ", a / b)     # always returns Float64
println("a ÷ b = ", a ÷ b)     # integer division (also: div(a, b))
println("a % b = ", a % b)     # remainder (also: rem(a, b))
println("a ^ b = ", a ^ b)     # exponentiation (not bitwise XOR)

# Unary minus
println(-a)

# Numeric literal coefficients: 2x means 2 * x
x = 10
println(2x + 3)
